src/App.js

```diff
@@ -1,24 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import "./styles/App.css";
+import Home from "./pages/home";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { About } from "./pages/about";
+import { Documentation } from "./pages/documentation";
```

Comment:
You can group these imports in one index file .
Hence, it wud look like this-> 
```import { About ,home,documentation} from "./pages";```

Reply:
Yup! I fixed the imports for components and pages as well.